pkg/genisoimage/genisoimage_test.go

```diff
@@ -0,0 +1,54 @@
+package genisoimage
+
+import (
+	"errors"
+	"fmt"
+	"testing"
+
+	"github.com/golang/mock/gomock"
+	. "github.com/onsi/ginkgo/v2/dsl/core"
+	. "github.com/onsi/gomega"
+	"github.com/openshift/appliance/pkg/executer"
+)
+
+var _ = Describe("Test GenIsoImage", func() {
+	var (
+		ctrl          *gomock.Controller
+		mockExecuter  *executer.MockExecuter
+		fakeCachePath = "/path/to/cache"
+		fakeDataPath  = "/path/to/data"
+		fakeImageName = "testdata.iso"
+	)
+
+	BeforeEach(func() {
+		ctrl = gomock.NewController(GinkgoT())
+		mockExecuter = executer.NewMockExecuter(ctrl)
+	})
+
+	It("genisoimage NewGenIsoImage - success", func() {
```

Comment:
Maybe replace NewGenIsoImage with GenerateImage, as that's the func we're actually testing.
E.g. GenerateImage - success

Reply:
Done